crates/core_simd/src/masks/full_masks.rs

```diff
@@ -140,62 +140,6 @@ where
         unsafe { Mask(core::intrinsics::simd::simd_cast(self.0)) }
     }
 
-    #[inline]
-    #[must_use = "method returns a new vector and does not mutate the original value"]
-    pub fn to_bitmask_vector(self) -> Simd<u8, N> {
-        let mut bitmask = Simd::splat(0);
-
-        // Safety: Bytes is the right size array
-        unsafe {
-            // Compute the bitmask
-            let mut bytes: <LaneCount<N> as SupportedLaneCount>::BitMask =
-                core::intrinsics::simd::simd_bitmask(self.0);
```

Comment:
This intrinsic still has some other uses... are those working fine with non-power-of-2?

Reply:
Answered [here](https://github.com/rust-lang/miri/pull/3655#issuecomment-2156105956):

> We still use the intrinsic, but currently (not yet synced to rust-lang/rust) we only use integer return types